setup.py

```diff
@@ -19,7 +19,7 @@
     long_description_content_type="text/markdown",
     include_package_data=True,
     url="https://github.com/Aura-healthcare/ecg_qc",
-    packages=setuptools.find_packages("ecg_qc/*", exclude=["tests"]),
+    packages=['ecg_qc'],
```

Comment:
Reopening this after a long time ... Thank you [adgilbert](https://github.com/adgilbert) for your PR!

Just one comment :
packages=setuptools.find_packages(exclude=["tests"]) would fix and keep the test exclusion.